test/app-tap/ssl-cert-paths-discover.test.lua

```diff
@@ -36,18 +36,6 @@ fio.copyfile(cert_file1, cert_file2)
 
 ----- Helpers
 
-local function cert_dir_paths_mockable()
```

Comment:
(If you'll update the patch series one more time, please mark the last commit as part of the issue too.)

Reply:
Updated.